lib/galaxy/model/__init__.py

```diff
@@ -204,6 +204,10 @@ def set_disk_usage( self, bytes ):
 
     total_disk_usage = property( get_disk_usage, set_disk_usage )
 
+    def adjust_total_disk_usage( self, amount ):
+        from sqlalchemy import func
```

Comment:
Yep.  That said, I'm really more focused on the reproduction and testing at this point -- if you have any ideas about that it'd be super helpful.